src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -476,7 +476,7 @@ public string Name
         public ObservableCollection<PortModel> InPorts
         {
             get { return inPorts; }
-            [IsObsolete("Property setter will be deprecated in Dynamo 3.0")]
+            [IsObsolete("Property setter will be deprecated in Dynamo a future version of Dynamo")]
```

Comment:
please try to convert all similar APIs to internal

Reply:
Yes, still used internally.

Reply:
Removed.